data/knexfile-config.js

```diff
@@ -0,0 +1,6 @@
+const knex=require('knex');
+const config=require('../knexfile.js');
+const environment=process.env.DB_environment || 'development';
```

Comment:
Could you change the env name to DB_ENV ? for env variables you want to always use CAPS.